src/test/resources/features/ManageFE/DraftDeclaration.feature

```diff
@@ -0,0 +1,34 @@
+@manageDraftDeparture
+Feature: Journey to test a draft departure in the Manage Transit Movements frontend
+
+  Scenario: 01 User starts a departure declaration, signs out, then returns to continue the declaration on a subsequent login before deleting the draft
+    Given I login with ID 1234567890
```

Comment:
Given I login with ID 1234567890
    When I click on the Make a departure declaration link on the 'Manage your transit movements' page

Both lines can be moved to the background block

Reply:
There is only 1 scenario so using 'Background' is not really appropriate as it is meant to save repetition when you have a few scenarios in a feature file.